scripts/proto_to_js.js

```diff
@@ -0,0 +1,15 @@
+var lines = [];
+var lineReader = require('line-reader');
+
+lineReader.eachLine('node_modules/grpc-bus/proto/grpc-bus.proto', function(line, last) {
```

Comment:
Oh, no -- it's 1. relying on file location within the dependent lib (which isn't officially exported as a public API) and 2. doing the simple regex/string-based escaping, which seems likely to not cover all corner cases.

Like i said, though, this is just a helper script and isn't something that's going to break in production, so if it's working that's good enough